backend/settings.py

```diff
@@ -135,3 +131,9 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+# CORS Headers
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:8000",
+    "http://localhost:8001",
```

Comment:
- Why 8001 ?
- shouldn't we add the port on which front end is running as well ?

Reply:
When I serve the back on port 8000 (default behavior), the front is automaticly served on port 8001, so I allowed both.

Reply:
We can define a port for the frontend ? Something like 3000 or 5000 is better I think.

Reply:
Okay, let's go for port 5000 then. Should I add it to `package.json` like this :
```
"scripts": {
        "dev": "vite --host localhost --port 5000",
        ...
}
```

?